scripts/recover_missing_archival_data.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+set -eox pipefail
+
+# This script can be used on archival nodes to recover missing data from early 2024.
+#
+# Requirements:
+# - The cold store must be mounted on an SSD to make the operation faster.
+# - The config 'resharding_config.batch_delay' must be set to 0.
+# - neard service must be stopped for the entire duration.
+# 
+# The script is idempotent. If interrupted, it can be restarted from the beginning, 
+# although in the interest of time it's advised to continue from the last successful step.
+#
+# On top of every command, where applicable, we include the number of expected database writes.
+# The estimated number of batches assumes a batch size of 500.0 KB. 
+#
+# The entire procedure is expected to take several days to complete.
+
+##########
+# neard must be stopped
+##########
+systemctl is-active --quiet neard && echo "neard must be stopped" && exit 1
```

Comment:
This is how we run neard in nearone, it doesn't necessarily need to be the same for other node operators.

Reply:
Good point

it's just for convenience - if you run the commands with a running `neard` you end up getting `error DB already locked`